src/report/superset-reports.factory.js

```diff
@@ -36,25 +36,25 @@
         if (SUPERSET_URL.substr(0, 2) !== '${') {
             reports = {
                 REPORTING_RATE_AND_TIMELINESS: createReport('reportingRateAndTimeliness',
-                    SUPERSET_URL + '/superset/dashboard/1/',
+                    SUPERSET_URL + '/superset/dashboard/taxa-de-submissao-de-relatorios/',
```

Comment:
I am not sure if those numbers should be changed

Reply:
That is right, I have overlooked it. In Angola project, we are using it in another way. I have reverted the version with numbers. Thank you @lukaslew.